platform/android/src/style/android_conversion.hpp

```diff
@@ -110,11 +110,32 @@ class ConversionTraits<mbgl::android::Value> {
     }
 
     static optional<GeoJSON> toGeoJSON(const mbgl::android::Value &value, Error &error) {
-        if (value.isNull() || !value.isString()) {
+        if (value.isNull()) {
```

Comment:
Could we omit this check? Isn't it so that `value.isString() || value.isObject()` means `!isNull()`?

Reply:
No, we can't. If value is null and call `vaule.isString()` in this line https://github.com/mapbox/mapbox-gl-native/blob/44ac1f008aeda27d13e6390405a3666ff522611b/platform/android/src/style/value.cpp#L30 `value.get()` will cause null point error.